hree01-sp2

hree01 opened the pull request:
Odevzdávám pilotní verzi sp2

hree01 committed:
hree01-sp2

Odevzdávám pilotní verzi sp2

hree01 committed:
version 02

Finální verze před obhajobou (původně omylem odevzdáno do jiného Pull Requestu).

nvbach91 commented:
Link to application: https://eso.vse.cz/~hree01/sp2/

nvbach91 reviewed (changes requested):
- vyhledavaci formular s jejich prvky musi byt v elementu form a zachycenou udalosti submit
- chybi indikace nacitani dat pri vyberu hlavnich mest v levem panelu
- klikatelna tlacitka musi mit k tomu prizpusobeny kurzor
- v sekci favorite cities se nesmi duplikovat informace o pocasi

![image](https://github.com/user-attachments/assets/47444c0a-2e32-4f13-8152-c1f9a2dea265)

- elementy z HTML nevybirat opakovane v prubehu programu, ale ukladat do promennych
- nezasahovat do DOM v cyklu (pridavani novych elementu, zmena atributu, obsahu, ...)

nvbach91 commented:
- do oblibenych ukladat i stat pro rozliseni mest se stejnymi nazvy

hree01 committed:
version 03

OPRAVA:
vyhledavaci formular s jejich prvky musi byt v elementu form a zachycenou udalosti submit 
chybi indikace nacitani dat pri vyberu hlavnich mest v levem panelu
klikatelna tlacitka musi mit k tomu prizpusobeny kurzor
v sekci favorite cities se nesmi duplikovat informace o pocasi
elementy z HTML nevybirat opakovane v prubehu programu, ale ukladat do promennych
nezasahovat do DOM v cyklu (pridavani novych elementu, zmena atributu, obsahu, ...)
+
do oblibenych ukladat i stat pro rozliseni mest se stejnymi nazvy

hree01 reviewed (commented):
Zapomněla jsem přidat upravenou verzi do složky www. Odevzdám znovu

hree01 committed:
real version 03 

version 03
OPRAVA:
vyhledavaci formular s jejich prvky musi byt v elementu form a zachycenou udalosti submit 
chybi indikace nacitani dat pri vyberu hlavnich mest v levem panelu
klikatelna tlacitka musi mit k tomu prizpusobeny kurzor
v sekci favorite cities se nesmi duplikovat informace o pocasi
elementy z HTML nevybirat opakovane v prubehu programu, ale ukladat do promennych
nezasahovat do DOM v cyklu (pridavani novych elementu, zmena atributu, obsahu, ...)
+
do oblibenych ukladat i stat pro rozliseni mest se stejnymi nazvy
